package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cordova-plugin-advanced-imagepicker",
-  "version": "1.5.5",
```

Comment:
Remove these changes completely. This is done automatically!

Reply:
It was a mistake, sorry.